ChangeLog.md

```diff
@@ -18,6 +18,8 @@ Bug fixes:
 * Run the Cabal file checking in the `sdist` command more reliably by
   allowing the Cabal library to flatten the
   `GenericPackageDescription` itself.
+* Script interpreter implicit file argumenst are now passed before other
```

Comment:
Typo here in arguments - also, this might not be obvious to someone coming here to find why something broke (if that might happen) - perhaps you can say 'like +RTS' or something like that, from the use case in #3658.

Reply:
Good catch! Fixed